src/Aws/EventBridgeClient.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Common\Aws;
+
+use Aws\EventBridge\EventBridgeClient as AWSEventBridgeClient;
+
+/**
+ * Class EventBridgeClient
+ * @package Common\Aws
+ */
+class EventBridgeClient extends AWSEventBridgeClient
+{
+    public function publishEvent(string $type, array $data, array $entryParams = [])
+    {
+        $meta = [];
+        if (class_exists('DDTrace\\GlobalTracer')) {
+            $meta = [
+                'meta' => [
+                    'DD_TRACE_ID' => \DDTrace\trace_id(),
+                    'DD_SPAN_ID' => \dd_trace_peek_span_id(),
+                ]
+            ];
+        }
+
+        $event = [
+            'Entries' => [
+                array_merge([
+                    'DetailType' => $type,
+                    'Detail' => json_encode(array_merge($data, $meta)),
+                    'EventBusName' => getenv('AWS_EVENT_BUS'),
```

Comment:
Wouldn't this mean that one project - can only use one eventbus ?

Reply:
unless it's set in the `$entryParams` - so i guess the idea is that if it's not overridden in the `$entryParams` then it'll get it from the env

Reply:
We're generally going to use single event bus for all the communication, so there shouldn't be the case where we need to change it